cli/src/config/templates.toml

```diff
@@ -529,39 +588,37 @@ coalesce(
 '''
 
 builtin_op_log_node = '''
-coalesce(
-  if(current_operation, label("current_operation", "@")),
-  "○",
-)
+if(current_operation, label("current_operation", "@"), "○")
 '''
 
 builtin_op_log_node_ascii = '''
-coalesce(
-  if(current_operation, label("current_operation", "@")),
-  "o",
-)
+if(current_operation, label("current_operation", "@"), "o")
 '''
```

Comment:
nit: leave `coalesce()`? I think these use `coalesce()` for ease of modification. I don't feel strongly, though.

Reply:
Done, reverted.